### ISSUE TITLE
Maximum Sub array problem

### ISSUE THREAD

**Issue by pratikpalashikar** [pratikpalashikar/leetcode-solutions#4]
Add the solution to the maximum sub array problem. And update the readme as well. The solution should be in java.

**Comment by pratikpalashikar** [comment]
Also please update the readme as well

**Comment by ronayumik** [comment]
Hi Pratik!
Is there any format for adding readme?

Thank you!

**Comment by pratikpalashikar** [comment]
Yeah mention problem name with link to the problem. If you see readme. You will come to know.
